config/routes.rb

```diff
@@ -10,12 +12,16 @@
     get 'signup', to: 'devise/registrations#new'
   end
 
-  resources :deliverers, only: %i[new create edit update]
-  resources :shifts, only: %i[new create edit update]
+  resources :deliverers, only: %i[index new create edit update]
+  resources :shifts, only: %i[index new create edit update]
 
-  get '/home', to: 'pages#home'
+  get '/assignments/show', to: 'assignments#show'
+
+  get '/assignments/new', to: 'assignments#new'
+
+  post '/assignments/search', to: 'assignments#search'
 
-  post '/assignments', to: 'assignments#create'
+  post '/assignments/create', to: 'assignments#create'
```

Comment:
we could use `resources` like deliverers and shifts above

Reply:
Fixed